lib/tasks/index.js

```diff
@@ -110,64 +61,35 @@ module.exports = function(ss, router, options) {
     },
 
     forget: function() {
-      for(var n in ss.orchestrator.tasks) {
-        delete ss.orchestrator.tasks[n];
+      for(var n in orchestrator.tasks) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.